packages/site/src/data/react/courses.ts

```diff
@@ -15,19 +15,21 @@ export const courseTags = [
 	"bootstrap",
 	"react context",
 	"hooks",
-	"typescript",
+	"TypeScript",
 	"tailwind",
-	"material UI",
+	"Material UI",
 	"vite",
-	"javascript marathon",
+	"JavaScript marathon",
 	"AWS",
 	"netlify",
 	"continuous integration",
-	"gatsbyJS",
+	"GatsbyJS",
 	"SSG",
 	"component design",
 	"apollo",
 	"faunaDB",
```

Comment:
should this one be "FaunaDB"?

Reply:
Yeah, it should be because it fits with what we laid out in the contributing docs.